src/App.svelte

```diff
@@ -5,20 +5,24 @@
 	import { onMount } from "svelte";
 	import { authState, loggedIn } from "./lib/store/authState";
 	import { onLogin, onLoggout } from "./lib/supabase.js";
+	import toast, { Toaster } from 'svelte-french-toast';
 
 	onMount(() => {
 		supabase.auth.onAuthStateChange((event, session) => {
 			if (event === 'SIGNED_IN' && !$loggedIn) {
 				$loggedIn = true;
+				toast.success("Logged in");
 				console.log('logged in');
 				try {
 					onLogin();
 				} catch (error) {
+					toast.error("Something went wrong!")
```

Comment:
Add this to the onLogin function

Reply:
How would I go on about adding an error to the onLogin function?

Reply:
Great question, just leave it there for now, I guess 👍